### PR TITLE
Nicer error msg if stdlib pkg with cgo flag is not installed

### PR DESCRIPTION
**Pull request by ArtfulCoder**
@thockin PTAL

### CODE REVIEW

File: hack/lib/golang.sh [hunk 1]
```diff
@@ -242,6 +242,27 @@ kube::golang::place_bins() {
   done
 }
 
+kube::golang::exit_if_stdlib_not_installed() {
+  local go_root_dir=`go env GOROOT`;
```
Comment (thockin) [inline hack/lib/golang.sh]: use $(go env GOROOT) syntax

**Comment by thockin** [comment]
LGTM except syntax.

**Comment by ArtfulCoder** [comment]
changed syntax. PTAL

**Comment by vmarmol** [comment]
LGTM, please squash commits and we'll merge.

**Comment by ArtfulCoder** [comment]
@vmarmol commits squashed!

**Comment by vmarmol** [comment]
Thanks @ArtfulCoder! Merging on green

**Comment by ArtfulCoder** [comment]
Thanks Victor!

**Pull request merged**